model/task_queue.go

```diff
@@ -372,24 +382,61 @@ func updateTaskQueue(distro string, taskQueue []TaskQueueItem, distroQueueInfo D
 	return errors.WithStack(err)
 }
 
-func ClearTaskQueue(distro string) error {
+func ClearTaskQueue(distroId string) error {
```

Comment:
Can we get a doc comment for the difference between `ClearTaskQueue` and `RemoveTaskQueue`?